content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
+API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
+rollout to a new release of Kubernetes).
+
+This mechanism is called the _Unknown Version Interoperability Proxy_.
+## Enabling the Unknown Version Interoperability Proxy
+The Unknown Version Interoperability Proxy feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate Unknown Version Interoperability Proxy by enabling
+the feature gate named `UnknownVersionInteroperabilityProxy` when you start the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
+
+```shell
+kube-apiserver \
+--feature-gates=UnknownVersionInteroperabilityProxy=true \
+# required flag for this feature
+--peer-ca-file=`ca bundle to verify serving certs of kube-apiserver`
+# optional flags for this feature
+--peer-advertise-ip=`ip of this kube-apiserver that should be used by peers to proxy requests`
+--peer-advertise-port=`port of this kube-apiserver that should be used by peers to proxy requests`
+# …and other flags as usual
+```
+
+## Interoperability proxying
+
+When you enable interoperability proxying, the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) 
+loads a special filter that identifies API groups / versions / resources that the local server doesn't recognise,
+and tries to proxy those requests to a peer API server that is capable of handling the request.
```

Comment:
nit: Most users don't care (and shouldn't need to care) how the proxying happens in the guts of the apiserver. Consider instead focusing on what happens to the request?  "When a request reaches an API server that cannot serve that API (either because it is at a version pre-dating the introduction of the API or the API is turned off...) the API server attempts to send the request to a peer...." ?

Reply:
I added a small description on what happens to the request, and another "how it works under the hood" section highlighting more details on how the feature works.